sktime/forecasting/ttm.py

```diff
@@ -216,6 +216,7 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
         # ajati, wgifford, vijaye12 for ibm-granite code
         "maintainers": ["geetu040"],
         "python_dependencies": ["transformers", "torch", "accelerate>=0.26.0"],
+        "python_version": ">=3.11",
```

Comment:
is this correct? Or does the original code run run on all python versions?

This is quite restrictive, so maybe you want to work on this in another PR?